Backend/ServiceLayer/Services/PointService.cs

```diff
@@ -37,7 +46,15 @@ public Point GetPoint(DatabaseContext _db, Guid Id)
 
         public Point UpdatePoint(DatabaseContext _db, Point point)
         {
-            return _PointRepo.UpdatePoint(_db, point);
+            try
+            {
+                var pointUpdated = _PointRepo.UpdatePoint(_db, point);
+                return pointUpdated;
+            }
+            catch (ArgumentOutOfRangeException e)
+            {
+                throw new InvalidPointException("Longitude/Latitude value invalid.", e);
+            }

```

Comment:
Catches the exception from the lower layer and wraps it to the point specific error to make error handling easier.